cms/envs/bok_choy.py

```diff
@@ -77,6 +77,15 @@
 YOUTUBE['TEST_URL'] = "127.0.0.1:{0}/test_youtube/".format(YOUTUBE_PORT)
 YOUTUBE['TEXT_API']['url'] = "127.0.0.1:{0}/test_transcripts_youtube/".format(YOUTUBE_PORT)
 
+FEATURES['ENABLE_COURSEWARE_INDEX'] = True
+SEARCH_ENGINE = "search.tests.mock_search_engine.MockSearchEngine"
+# Path at which to store the mock index - can remove disable=invalid-name
+# once global settings are set to disable this error and
+# disable=no-value-for-parameter follow the pattern above
```

Comment:
You say "can remove disable=invalid-name once global settings are set to disable this error and disable=no-value-for-parameter follow the pattern above". I assumed you were talking about adding these disables to the global `pylintrc` file, which I don't think is a good idea, but it's better than saying "we can make this change when that happens". (If it never happens, these comments will just stick around in this file forever, and we'll very quickly forget the context for them.)

Or are you referring to something else? If so, you probably want to re-word this comment to make it clearer.